include/SSVOpenHexagon/Global/Assets.hpp

```diff
@@ -137,19 +137,24 @@ class HGAssets
     void loadMusicData(const std::string& mPackId, const ssvufs::Path& mPath);
     void loadStyleData(const std::string& mPackId, const ssvufs::Path& mPath);
     void loadLevelData(const std::string& mPackId, const ssvufs::Path& mPath);
-    void loadCustomSounds(
-        const std::string& mPackId, const ssvufs::Path& mPath);
-    void loadLocalProfiles();
-
-    void saveCurrentLocalProfile();
+    void loadCustomSounds(const std::string& mPackId, const ssvufs::Path& mPath);
 
     const MusicData& getMusicData(
         const std::string& mPackId, const std::string& mId);
     const StyleData& getStyleData(
         const std::string& mPackId, const std::string& mId);
 
+    std::string reloadLevelData(const std::string& mPackId, const std::string& mPath, const std::string& mId);
+    std::string reloadMusicData(const std::string& mPackId, const std::string& mPath, const std::string& mId);
+    std::string reloadStyleData(const std::string& mPackId, const std::string& mPath, const std::string& mId);
+    std::string reloadMusic(const std::string& mPackId, const std::string& mPath, const std::string& mId);
+    std::string reloadCustomSounds(const std::string& mPackId, const std::string& mPath, const std::string& mId);
```

Comment:
In general, functions whose return value should never be ignored should be marked as `[[nodiscard]]`. Despite the verbosity, this allows the compiler to produce a warning if someone forgets to check the return value. It's particularly useful here because the names do not suggest that a return value will be returned:

```suggestion
    [[nodiscard]] std::string reloadLevelData(const std::string& mPackId, const std::string& mPath, const std::string& mId);
    [[nodiscard]] std::string reloadMusicData(const std::string& mPackId, const std::string& mPath, const std::string& mId);
    [[nodiscard]] std::string reloadStyleData(const std::string& mPackId, const std::string& mPath, const std::string& mId);
    [[nodiscard]] std::string reloadMusic(const std::string& mPackId, const std::string& mPath, const std::string& mId);
    [[nodiscard]] std::string reloadCustomSounds(const std::string& mPackId, const std::string& mPath, const std::string& mId);
```